server/services/transcation-service.js

```diff
@@ -0,0 +1,114 @@
+const Transaction = require('../models/transaction-model');
+const ReadPreference = require('mongodb').ReadPreference;
+
```

Comment:
we need to probably delete this file? (the typo in the name) I think the imports are fixed in other places so we should be fine to do that :)

Reply:
(we have transcation service and transaction service lol)

Reply:
Lol I made the same review lmao

Reply:
Deleted it